install/operator/pkg/syndesis/configuration/configuration_test.go

```diff
@@ -17,10 +17,11 @@ func TestStandardConfig(t *testing.T) {
 	}
 
 	config := GetEnvVars(&syndesis)
-	assert.Len(t, config, 3)
+	assert.Len(t, config, 4)
```

Comment:
The tests were failing without this (and the line below), but I'm not sure why it's these four env vars that need to be specified, when there's lots of others in configuration.go.

Reply:
Stab in the dark: this might be because of the default value `EnvDatavirtEnabled` has

Reply:
yep, that was it, I tried not setting the default value but then I would run into #6314 when komodo was set to disabled.